tests/ErrorHandler/HtmlRendererTest.php

```diff
@@ -34,4 +40,75 @@ public function testVerboseOutput(): void
         $this->assertStringContainsString(RuntimeException::class, $renderedOutput);
         $this->assertStringContainsString($exceptionMessage, $renderedOutput);
     }
+
+    public function testNonVerboseOutputWithCustomTemplate(): void
+    {
+        $templateFileContents = '<html><?php echo $throwable->getMessage();?></html>';
+        $this->createTestTemplate(self::CUSTOM_TEMPLATES['error'], $templateFileContents);
+
+        $renderer = new HtmlRenderer(self::CUSTOM_TEMPLATES);
+        $request = new ServerRequest('GET', '/', ['Accept' => ['text/html']]);
+        $renderer->setRequest($request);
+
+        $exceptionMessage = 'exception-test-message';
+        $exception = new \RuntimeException($exceptionMessage);
+
+        $renderedOutput = $renderer->render($exception);
+        $this->assertStringContainsString($exceptionMessage, $renderedOutput);
+        $this->assertStringContainsString('<html>', $renderedOutput);
```

Comment:
You have full content of template above. 
Please make sure that `$renderOutput` constain full templates artifacts